packages/gatsby-theme-carbon/src/components/SquareCard/square-card.scss

```diff
@@ -0,0 +1,185 @@
+// Square Card Group
+.square-card-group .#{$prefix}--col-lg-4 {
+  margin-bottom: 0;
+}
+.square-card-group .#{$prefix}--square-card {
+  border-top: 1px solid #e0e0e0;
+
+  @include carbon--breakpoint('md') {
+    border-left: 1px solid #e0e0e0;
+  }
+}
+.square-card-group .#{$prefix}--col-lg-4 .#{$prefix}--square-card {
```

Comment:
Do we need all this specificity? Could it just be `.#{$prefix}--square-card` (or `.squareCard` with CSS modules)?